### ISSUE TITLE
Warning: dependencies.dependency.version for javax.servlet-api is missing

### ISSUE THREAD

**Issue by nikolavojicic** [mattbertolini/spring-annotated-web-data-binder#11]
I'm getting this warning:
```
> Task :processResources UP-TO-DATE
> Task :classes UP-TO-DATE
Errors occurred while build effective model from C:\Users\nikol\.gradle\caches\modules-2\files-2.1\com.mattbertolini\spring-webmvc-annotated-data-binder\0.5.0\7704370e554c6f598147467c65471061a0d4909f\spring-webmvc-annotated-data-binder-0.5.0.pom:
    'dependencies.dependency.version' for javax.servlet:javax.servlet-api:jar is missing. in com.mattbertolini:spring-webmvc-annotated-data-binder:0.5.0

> Task :Application.main()
```

I've tried to stifle it by specifying the version but it didn't work:
```
implementation 'javax.servlet:javax.servlet-api:4.0.1'
```

Exclusion didn't help either.

**Comment by mattbertolini** [comment]
Hi @nikolavojicic. This has not appeared in my testing before so thanks for the report.

Could I ask you a few questions?

* What version of Spring are you using with the lib?
* Are you using Spring Boot? If so, what version?
* What version of Gradle are you using to build?
* I'm doubting this but I'll ask anyway 😄 is this a war file deployment?

I think the issue has to do with the fact that I am attempting to populate the version via a parent Gradle file using a platform. That may not work when it comes to publishing the POM file to Maven Central. I've made a bunch of changes to the build since the 0.5.0 release. Maybe some of those changes will help populate this correctly. 

I've published 0.6.0-SNAPSHOT to the Maven Central Snapshot Repository (https://oss.sonatype.org/content/repositories/snapshots/). If possible, could you try pulling that down and seeing if the error goes away. Based on the changes to the generated POM, I think they might. If it does fix the issue, I can push the 0.6.0 release. The code hasn't changed, just a restructuring of the build.

**Comment by nikolavojicic** [comment]
Thanks for the fast feedback!
- `'org.springframework.boot' version '2.6.4'`
- `gradle-7.4.1`
- Yes, it is a `war` file deployment
- `0.6.0-SNAPSHOT` doesn't emit that warning, it's fixed

**Comment by mattbertolini** [comment]
I'm glad the snapshot fixed the warning. I've published version 0.6.0 to Maven Central. Thanks for reporting it.
